src/components/backend-ai-console.ts

```diff
@@ -329,6 +340,13 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
       });
   }
 
+  _initClient() {
```

Comment:
How about using `globalThis.backendaiclient` instead of spawning another instance?

Reply:
I found out that 'client' property isn't used in this component. therefore It should be removed.